### PR TITLE
Added KITT4SME model

### PR DESCRIPTION
**Pull request by ntonfoholonix**

### CODE REVIEW

**Comment by albertoabellagarcia** [comment]
Thanks for the contribution. The syntax of the schema is correct but the descriptions are not complete. 
Check this [link](https://smartdatamodels.org/extra/check_schema.php?schemaUrl=https%3A%2F%2Fraw.githubusercontent.com%2Fsmart-data-models%2Fincubated%2F4bcad727b4c28123f935f4f42577d2ae7626ac2c%2FSMARTMANUFACTURING%2FKITT4SME%2Fschema.json&mail=alberto.abella%40fiware.org&test=1&yamlOutput=0)
Where you have 3 sections 
- documentationStatusOfProperties 
- schemaDiagnose
- metadata
Where you have missing descriptions.
You can also review the [contribution manual section](https://docs.google.com/presentation/d/e/2PACX-1vTs-Ng5dIAwkg91oTTUdt8ua7woBXhPnwavZ0FxgR8BsAI_Ek3C5q97Nd94HS8KhP-r_quD4H0fgyt3/pub?start=false&loop=false&delayms=3000#slide=id.gcc1dca0563_0_16) to be sure.
And of course, you can use the [documentation checking service](https://smartdatamodels.org/index.php/data-models-contribution-api/) it is the same first link in this comment.

File: SMARTMANUFACTURING/KITT4SME/ADOPTERS.yml
Comment (albertoabellagarcia) [inline SMARTMANUFACTURING/KITT4SME/ADOPTERS.yml:1]: The model is not KITTS4SME but Wearable, isn't it? It should be changed, then.

File: SMARTMANUFACTURING/KITT4SME/examples/example.json
Comment (albertoabellagarcia) [inline SMARTMANUFACTURING/KITT4SME/examples/example.json:1]: naming is 'wrong' this file should be example-normalized.json. 
The schema.json has to validate example.json and example.jsonld which are the keyvalues export from the normalized format (like this one).

File: SMARTMANUFACTURING/KITT4SME/schema.json
Comment (albertoabellagarcia) [inline SMARTMANUFACTURING/KITT4SME/schema.json:1]: You have created (and this is quite valuable) a schema for the validation of the payload in normalized format, but we only create the schemas for validating the keyvalues format, therefore there is one much simpler than this to be done. 
this is the key value example corresponding to your example
{
  "id": "urn:ngsi:Wearable:001",
  "type": "Wearable",
  "wearableDevice": {
    "workerSession": {
      "session": {
        "start": {
          "dateTime": "2022-10-12T15:14:12.348Z",
          "format": "YYYY-MM-DDThh:mm:ss.zzzZ",
          "timezoneId": "UTC"
        },
        "end": {
          "dateTime": "2022-10-12T18:01:10.120Z",
          "format": "YYYY-MM-DDThh:mm:ss.zzzZ",
          "timezoneId": "UTC"
        }
      },
      "worker": {
        "workerId": "8c31dcdd-a546-4583-8131-9adf6d97ca74",
        "description": ""
      }
    },
    "wearableAccelometer": {
      "x": 2.8,
      "y": 1.2,
      "z": -1.5
    },
    "name": "Empatica E4",
    "deviceProperties": {
      "properties": {}
    },
    "description": ""
  },
  "workerId": "8c31dcdd-a546-4583-8131-9adf6d97ca74",
  "sessionId": "f54354a4-e998-4de7-9a40-6d46d7f37346",
  "temperature": 37,
  "gsr": 1.4,
  "heartRate": 80,
  "rrInterval": 0,
  "position": {
    "latitude": 0,
    "longitude": 0
  },
  "timestamp": "2022-10-12T16:34:56.718Z"
}

Comment (albertoabellagarcia) [reply SMARTMANUFACTURING/KITT4SME/schema.json:1]: In fact thanks to some tolls I guess that a candidate for the schema would be this one. Please review and validate
{
  "$schema": "http://json-schema.org/schema#",
  "$schemaVersion": "0.0.1",
  "modelTags": "KITT4SME, Sensing Layer, Holonix, EU, Project, Manufacturing, Worker, Session",
  "derivedFrom": "",
  "license": "",
  "$id": "https://smart-data-models.github.io/XXXsubjectXXX/XXXdataModelXXX/schema.json",
  "title": "KITT4SME Sensing Layer",
  "description": "Data model produced by the Sensing Layer in the KITT4SME project",
  "type": "object",
  "required": [
    "id",
    "type"
  ],
  "allOf": [
    {
      "$ref": "https://smart-data-models.github.io/data-models/common-schema.json#/definitions/GSMA-Commons"
    },
    {
      "$ref": "https://smart-data-models.github.io/data-models/common-schema.json#/definitions/Location-Commons"
    },
    {
      "properties": {
        "id": {
          "type": "string",
          "pattern": "^urn:ngsi:Wearable:[0-9]+$"
        },
        "type": {
          "type": "string",
          "enum": [
            "Wearable"
          ]
        },
        "wearableDevice": {
          "type": "object",
          "properties": {
            "workerSession": {
              "type": "object",
              "properties": {
                "session": {
                  "type": "object",
                  "properties": {
                    "start": {
                      "type": "object",
                      "properties": {
                        "dateTime": {
                          "type": "string",
                          "format": "date-time"
                        },
                        "format": {
                          "type": "string",
                          "pattern": "^YYYY-MM-DDThh:mm:ss.zzzZ$"
                        },
                        "timezoneId": {
                          "type": "string",
                          "enum": [
                            "UTC"
                          ]
                        }
                      }
                    },
                    "end": {
                      "type": "object",
                      "properties": {
                        "dateTime": {
                          "type": "string",
                          "format": "date-time"
                        },
                        "format": {
                          "type": "string",
                          "pattern": "^YYYY-MM-DDThh:mm:ss.zzzZ$"
                        },
                        "timezoneId": {
                          "type": "string",
                          "enum": [
                            "UTC"
                          ]
                        }
                      }
                    }
                  }
                },
                "worker": {
                  "type": "object",
                  "properties": {
                    "workerId": {
                      "type": "string",
                      "format": "uuid"
                    },
                    "description": {
                      "type": "string"
                    }
                  }
                }
              }
            },
            "wearableAccelometer": {
              "type": "object",
              "properties": {
                "x": {
                  "type": "number"
                },
                "y": {
                  "type": "number"
                },
                "z": {
                  "type": "number"
                }
              }
            },
            "name": {
              "type": "string"
            },
            "deviceProperties": {
              "type": "object",
              "properties": {
                "properties": {
                  "type": "object"
                }
              }
            },
            "description": {
              "type": "string"
            }
          }
        },
        "workerId": {
          "type": "string",
          "format": "uuid"
        },
        "sessionId": {
          "type": "string",
          "format": "uuid"
        },
        "temperature": {
          "type": "number"
        },
        "gsr": {
          "type": "number"
        },
        "heartRate": {
          "type": "number"
        },
        "rrInterval": {
          "type": "number"
        },
        "position": {
          "type": "object",
          "properties": {
            "latitude": {
              "type": "number"
            },
            "longitude": {
              "type": "number"
            }
          }
        },
        "timestamp": {
          "type": "string",
          "format": "date-time"
        }
      }
    }
  ]
}

File: SMARTMANUFACTURING/KITT4SME/notes.yaml
Comment (albertoabellagarcia) [inline SMARTMANUFACTURING/KITT4SME/notes.yaml:1]: although you have no content to customize you need to include a templated version
https://github.com/smart-data-models/data-models/blob/master/templates/dataModel_for_submision/notes.yaml
and remove texts that do not apply to you

File: SMARTMANUFACTURING/KITT4SME/examples/example.json
Comment (albertoabellagarcia) [reply SMARTMANUFACTURING/KITT4SME/examples/example.json:1]: I guess that a candidate could be this one. Please check and validate
{
  "id": "urn:ngsi:Wearable:001",
  "type": "Wearable",
  "wearableDevice": {
    "workerSession": {
      "session": {
        "start": {
          "dateTime": "2022-10-12T15:14:12.348Z",
          "format": "YYYY-MM-DDThh:mm:ss.zzzZ",
          "timezoneId": "UTC"
        },
        "end": {
          "dateTime": "2022-10-12T18:01:10.120Z",
          "format": "YYYY-MM-DDThh:mm:ss.zzzZ",
          "timezoneId": "UTC"
        }
      },
      "worker": {
        "workerId": "8c31dcdd-a546-4583-8131-9adf6d97ca74",
        "description": ""
      }
    },
    "wearableAccelometer": {
      "x": 2.8,
      "y": 1.2,
      "z": -1.5
    },
    "name": "Empatica E4",
    "deviceProperties": {
      "properties": {}
    },
    "description": ""
  },
  "workerId": "8c31dcdd-a546-4583-8131-9adf6d97ca74",
  "sessionId": "f54354a4-e998-4de7-9a40-6d46d7f37346",
  "temperature": 37,
  "gsr": 1.4,
  "heartRate": 80,
  "rrInterval": 0,
  "position": {
    "latitude": 0,
    "longitude": 0
  },
  "timestamp": "2022-10-12T16:34:56.718Z"
}

**Review by albertoabellagarcia — COMMENTED** [review]
Please attend the comments and sorry for the delay

**Comment by ntonfoholonix** [comment]
@albertoabellagarcia  Thank you for your feedback. I've done and pushed some changes according to your osservations.

**Review by albertoabellagarcia — APPROVED** [review]
Thanks for the changes

**Pull request merged**